Composer/packages/client/src/components/TestController/TestController.tsx

```diff
@@ -59,7 +59,7 @@ export const TestController: React.FC<{ projectId: string }> = (props) => {
   const [calloutVisible, setCalloutVisible] = useState(false);
 
   const botActionRef = useRef(null);
-  const notifications = useNotifications(projectId);
+  const notifications = useRecoilValue(allDiagnosticsSelector);
```

Comment:
This change needs to be ported over because of this. Based on which PR gets merged first we can decide who does the port.

https://github.com/microsoft/BotFramework-Composer/pull/4498/files#diff-7490c82b785dc2387b962672c0217d1aee42428e238de52d30cb660d8bb380f2